src/radio/src/RadioGroup.js

```diff
@@ -99,14 +17,24 @@ export default class RadioGroup extends PureComponent {
       options,
       onChange,
       isRequired,
-      ...props
-    } = this.props
+      ...rest
+    } = props
+
+    const name = useRef()
+    name.current = `RadioGroup-${radioCount}`
+    radioCount += 1
```

Comment:
Wont this produce new names on re-renders?

Reply:
Ah yeah -- I'll move to a ref. Good spot.

Reply:
yeah -- good spot. should just update / set the ref on mount